conf/machine/include/imx-base.inc

```diff
@@ -480,6 +480,10 @@ PREFERRED_VERSION_libdrm:mx6-nxp-bsp ??= "2.4.109.imx"
 PREFERRED_VERSION_libdrm:mx7-nxp-bsp ??= "2.4.109.imx"
 PREFERRED_VERSION_libdrm:mx8-nxp-bsp ??= "2.4.109.imx"
 
+PREFERRED_VERSION_vulkan-headers:imxvulkan ??= "1.2.182.0"
+PREFERRED_VERSION_vulkan-loader:imxvulkan  ??= "1.2.182.0"
+PREFERRED_VERSION_vulkan-tools:imxvulkan   ??= "1.2.182.0"
+
```

Comment:
We need to move packages to SoC architecture when depending on those automatically. I think the following diff does the trick:

```diff
diff --git a/conf/machine/include/imx-base.inc b/conf/machine/include/imx-base.inc
index a0d6be13..dd1d8d5d 100644
--- a/conf/machine/include/imx-base.inc
+++ b/conf/machine/include/imx-base.inc
@@ -306,6 +306,11 @@ MACHINE_SOCARCH_FILTER:append:imxvpu = " \
     libimxvpuapi \
     virtual/imxvpu \
 "
+MACHINE_SOCARCH_FILTER:append:imxvulkan = " \
+    vulkan-headers \
+    vulkan-loader \
+    vulkan-tools \
+"
 MACHINE_SOCARCH_FILTER:append:imxgpu = " \
     virtual/egl \
     virtual/mesa \
```